spec/controllers/application_controller_spec.rb

```diff
@@ -184,7 +184,7 @@ def index
         allow(controller).to receive(:current_sp).and_return(sp)
 
         expect(Analytics).to receive(:new).
-          with(user: user, request: request, sp: sp.issuer, first_path_visit_this_session: true,
+          with(user: user, request: request, sp: sp.issuer, session: match_array({}),
```

Comment:
`match_array` passed with a hash seems unexpected? Could this just be...

```suggestion
          with(user: user, request: request, sp: sp.issuer, session: {},
```

Same with below.

Reply:
Was getting new with unexpected arguments yet no diff.  googled it and found that match_array fixed it

Reply:
Maybe we could `hash_including` or something? I think what @aduth was getting at is that it's weird to see array in the name but pass in a hash literal